testing/bsg_misc/bsg_idiv_iterative/divide_4.stim

```diff
@@ -0,0 +1,256 @@
+0 0
```

Comment:
I believe we said this file should have the hand-checked inputs and outputs side by side, and a comment that mentions that fact? This way, we can just review this file brute force by hand and have confidence that there is not a bug in our checking code.

Reply:
So the signed.c and unsigned.c programs takes in an input file (i.e. the file you pointed to. and it contains all the possible dividend and divisor outputs for the divider width you want to test) and then produces the file that you describe. Then, the output of the simulation is checked against said file. So a client need only produce a file with all the inputs they want to test and then running make and then make verify will create a file with the expected output (in the format you described) and then check the differences with the simulation output file (also in the format you described).

Reply:
I can easily write a README to clarify this.

Reply:
The idea behind the way I set it up is that the user does not need to know the expected output of their desired input cases, it can be produced for them. Additionally, I commented the RISC-V standards that define the corner cases in the C code that produces the file with expected output.